bot.js

```diff
@@ -9,7 +9,7 @@ if (process.env.TEST) {
   console.log(`[${process.env.USER} => ${config.name}]`);
 } else {
   const IrcClient = require('./interfaces/irc');
-  var client = new IrcClient(config.server, config.name, config.channels);
+  client = new IrcClient(config.server, config.name, config.channels);
```

Comment:
Yes that is a good approach. I feel `let` less confusing as well. I will update it and merge.